### PR TITLE
CAR-1124  boringssl also contains libdecrepit

### PR DESCRIPTION
**Pull request by artasano**
Please do not send pull requests to the BoringSSL repository.

We do, however, take contributions gladly.

See https://boringssl.googlesource.com/boringssl/+/master/CONTRIBUTING.md

Thanks!

### CODE REVIEW

File: decrepit/CMakeLists.txt [hunk 1]
```diff
@@ -3,13 +3,6 @@ include_directories(../include)
 add_library(
   decrepit
 
-  # TODO(Art): not sure if it is sill necessary
-  ## airtime: start
-  ## DEVOPS-1090: libdecrepit.a is not created by Xcode generator
-  ## unless it has a real source file (not just referencing TARGET_OBJECTS)
-  ## See https://gitlab.kitware.com/cmake/cmake/issues/16524
-  #airtime_dummy_file.c
-  ## airtime: end
```
Comment (artasano) [inline decrepit/CMakeLists.txt:12]: Unrelated cleanup.

**Review by addisonpolcyn — APPROVED** [review]

**Comment by jim-wong** [comment]
Is this causing problems?

**Comment by artasano** [comment]
ffmpeg uses rtmpdump and that uses libdecrepit.
I have some project with ffmpeg and getting link error.

**Review by jim-wong — APPROVED** [review]
